stdlib/Profile/src/Profile.jl

```diff
@@ -788,9 +794,34 @@ function flat(io::IO, data::Vector{UInt64}, lidict::Union{LineInfoDict, LineInfo
     return false
 end
 
+# make a terminal-clickable link to the file and linenum.
+# Similar to `define_default_editors` in `Base.Filesystem` but for creating URIs not commands
+function editor_link(path::String, linenum::Int)
```

Comment:
At a glance it looks like `path` is directly interpolated? This looks a little dodgy to me, what about a path like `/tmp/?file=redherring&line=42/file.jl`? Will that become say `idea://open?file=/tmp/?file=redherring&line=42/file.jl&line=1`

Reply:
That path isn't a file path? and especially not something that the profiler could return, as far as I understand...?
Though escaping the path generally seems like a good idea.

Also it'd be great to have https://github.com/JuliaLang/julia/pull/55454 finished to use in this function for the generic case.

Reply:
Ah I see what you're saying.. a dir named `?file=redherring&line=42`. Ok

Reply:
https://github.com/JuliaLang/julia/pull/55857

Reply:
> Also it'd be great to have https://github.com/JuliaLang/julia/pull/55454 finished to use in this function for the generic case.

From my perspective, that PR is good to go, just awaiting further comment or merging.